stl/inc/variant

```diff
@@ -956,7 +933,6 @@ template <class... _Types>
 using _Variant_destroy_layer = conditional_t<conjunction_v<is_trivially_destructible<_Types>...>,
     _Variant_base<_Types...>, _Variant_destroy_layer_<_Types...>>;
 
-// CLASS TEMPLATE variant
 #ifdef __clang__
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-volatile"

```

Comment:
There's at least one shouty banner below that's not being removed:

https://github.com/microsoft/STL/blob/6a53cf2b809cbd3c10d5f35226e996ad5486ac06/stl/inc/variant#L1256